src/native/corehost/fxr/fx_muxer.cpp

```diff
@@ -1107,7 +1107,7 @@ int fx_muxer_t::handle_cli(
 
     if (pal::strcasecmp(_X("--info"), argv[1]) == 0)
     {
-        command_line::print_muxer_info(host_info.dotnet_root, resolver.global_file_path());
+        command_line::print_muxer_info(host_info.dotnet_root, resolver.global_file_path(), true /*skip_sdk_info_output*/);
```

Comment:
Nit: Technically it would be better to print out everything if something in the SDK failed (something's broken and we can't run managed code for example):
```suggestion
        command_line::print_muxer_info(host_info.dotnet_root, resolver.global_file_path(), result == 0 /*skip_sdk_info_output*/);
```